lib/hoe.rb

```diff
@@ -536,6 +543,13 @@ def define_spec
       s.require_paths        = dirs unless dirs.empty?
       s.rdoc_options         = ["--main", readme_file]
       s.post_install_message = post_install_message
+      s.metadata             = {}.tap do |meta|
+        meta["bug_tracker_uri"] = urls["bugs"] if urls["bugs"]
+        meta["changelog_uri"] = urls["clog"] if urls["clog"]
+        meta["documentation_uri"] = urls["doco"] if urls["doco"]
+        meta["homepage_uri"] = urls["home"] if urls["home"]
+        meta["source_code_uri"] = urls["code"] if urls["code"]
```

Comment:
I think I'd rather see this as a 1 line loop around a mapping (eg from "bug_tracker_uri" to "bugs").

Reply:
Done